### PR TITLE
Build against maintained Node versions

### PR DESCRIPTION
**Pull request by jdesboeufs**
* Drop Node 0.10, 5 and 7
* Add Node 8

See #200 and #202

### CODE REVIEW

**Comment by jdesboeufs** [comment]
I'm not familiar with AppVeyor :(
Some part of @BergWerkGIS PR (#203) may help!

**Comment by jdesboeufs** [comment]
Now rebased on @BergWerkGIS PR (#203)

**Comment by brianreavis** [comment]
@BergWerkGIS @springmeyer This all looks good to me. How about you guys?

**Comment by springmeyer** [comment]
Looks great to me. @BergWerkGIS's work starts consolidating all the windows build needs into a single script. The goal is to reduce duplication and the challenge of keeping track of one-of fixes. I've added you as a commiter to https://github.com/mapbox/ci-scripts/ @brianreavis - in case you ever need to modify something there to keep node-gdal building smoothly.

**Comment by brianreavis** [comment]
Thanks! Just to confirm, is it fine on your end to drop 0.10 support with this release? Referencing https://github.com/naturalatlas/node-gdal/issues/202

If we drop it we should probably bump the minor version number.

**Comment by jdesboeufs** [comment]
For now, the next version would just bring prebuilt binaries for Node 8.
It will be a no-op version for other Node version.
So you can consider Node 0.10 support ends with `0.9.x` branch and bump the minor version.

**Comment by jdesboeufs** [comment]
Something is missing?
Poor Node 8 developers we are cannot support rebuild GDAL anymore 😹

**Comment by springmeyer** [comment]
> Thanks! Just to confirm, is it fine on your end to drop 0.10 support with this release? Referencing #202

Sorry for missing responding to this. My preference would be to add back v0.10.x support for one last release. The commits in @BergWerkGIS should work fine with node v0.10.x. My logic for wanting this is that I'm personally trying to upgrade a lot of downstream deps from node v0.10.x to v4/6 and being able to test with both - for one node-gdal release- would make this process less scary for me. Also if the version being only a patch release bump is important, because all node-gdal deps I maintain use `~major.minor.patch` in the package.json like https://github.com/mapbox/mapnik-omnivore/blob/7cfb58eb4f689e6a6330a05e01e7087f5cfb6cb9/package.json#L16. So a major or minor bump is painful since it requires updating a cascade of deps.

Long story short: can we do a `v0.9.5` with node v0.10.x support? @brianreavis please mention me if you need help with this or if you'd like me to run the release out.

**Comment by brianreavis** [comment]
@springmeyer Sure thing! I'll add it back in and put out a patch release today assuming it all builds fine.

**Comment by springmeyer** [comment]
Perfect! Ping me if you see failures and I'll jump on it.
